package.json

```diff
@@ -4,10 +4,18 @@
   "repository": "lukeed/qss",
   "description": "A tiny (305B) browser utility for stringifying a query Object.",
   "unpkg": "dist/qss.min.js",
-  "module": "dist/qss.m.js",
+  "module": "dist/qss.mjs",
   "main": "dist/qss.js",
   "types": "qss.d.ts",
   "license": "MIT",
+  "exports": {
+    ".": {
+      "types": "./qss.d.ts",
+      "import": "./dist/qss.mjs",
+      "require": "./dist/qss.js"
```

Comment:
without "default", it won't work in a number of node versions included in your engines declaration. can you add that?

Reply:
Every node resolver always either has import or require flagged. This is true for all node versions I’m aware of.

Reply:
It's not true for node 13.3 - 13.6 (node 13.0-13.2 only support the string form, but do support array fallbacks).

Your engines say `>= 4`, which explicitly includes these versions.

Reply:
You can peruse the test cases in https://www.npmjs.com/package/node-exports-info if you like :-)